studio/components/interfaces/Billing/Subscription/Subscription.tsx

```diff
@@ -41,10 +40,11 @@ const Subscription: FC<Props> = ({ showProjectName = false }) => {
     isLoading: loading,
     error,
   } = useProjectSubscriptionQuery({ projectRef: ui.selectedProject?.ref })
-  const { data: usage, isLoading: loadingUsage } = useProjectUsageQuery({ projectRef })
+  const { data: usage } = useProjectUsageQuery({ projectRef })
 
   const isPayg = subscription?.tier.prod_id === STRIPE_PRODUCT_IDS.PAYG
   const isEnterprise = subscription?.tier.supabase_prod_id === PRICING_TIER_PRODUCT_IDS.ENTERPRISE
+  const canOnlyUpdateAddons = isEnterprise
```

Comment:
Team tier should also be able to update addons

Reply:
updated!